### PR TITLE
Output target

### PR DESCRIPTION
**Pull request by gtaltemann**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
```
Comment (kewitz) [inline .babelrc]: You don't need to separate envs if they're all the same.

File: package.json [hunk 2]
```diff
@@ -2,16 +2,34 @@
   "description": "Invisible's Frontend Stack using Mobx, React and SAM Pattern",
   "name": "@invisible/samx",
   "version": "1.0.0",
-  "main": "src/index.js",
+  "private": true,
```
Comment (kewitz) [inline package.json]: This is not private.

File: package.json [hunk 3]
```diff
@@ -2,16 +2,34 @@
...
   "repository": "git@github.com:invisible-tech/samx.git",
   "author": "Invisible Technologies Inc.",
   "license": "MIT",
+  "scripts": {
+    "build": "./node_modules/.bin/parcel build -d ./dist"
+  },
```
Comment (kewitz) [inline package.json]: What are we building here?

File: src/index.js [hunk 4]
```diff
@@ -7,4 +7,4 @@ configure(config[process.env.NODE_ENV] || config.development)
 
 export { Model, Propose } from './model'
 export { State } from './state'
-export { Render } from './render'
+export { Render } from './render'
```
Comment (kewitz) [inline src/index.js]: Add blank line at the end of the file.

File: package.json [hunk 5]
```diff
@@ -2,16 +2,34 @@
...
+    "mobx-react": "^5.2.3",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1"
+  },
   "dependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
```
Comment (kewitz) [inline package.json]: It's a library, we don't need to build neither add babel.

File: package.json [hunk 6]
```diff
@@ -2,16 +2,34 @@
...
     "debug": "^3.1.0",
     "lodash": "^4.17.10",
     "mobx": "^5.0.3",
     "mobx-react": "^5.2.3",
+    "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "webpack": "^4.12.0"
+  },
+  "devDependencies": {
+    "webpack-cli": "^3.0.8"
```
Comment (kewitz) [inline package.json]: Remove webpack, we don't need to build anything here.

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+        use: {
+            loader: 'babel-loader'
+        }
+      }
+    ]
+  },
+  externals: {
+    react: 'commonjs2 react',
+    mobx: 'commonjs2 mobx',
+ },
```
Comment (kewitz) [inline webpack.config.js]: Why!?

File: .babelrc (hunk 1 above)
Comment (jshrt) [reply .babelrc]: I'll update you about this, I don't remember why we did this but it had to do with Babel + TS and I suppose it is like that in Midgard too.

File: package.json (hunk 3 above)
Comment (jshrt) [reply package.json]: We're using webpack instead, it seems that the package alone wouldn't work when included in Midgard, the other thing with building is that we'll need to build for node and browser.

**Comment by kewitz** [comment]
Fixed by #3.

**Pull request closed without merging**